content/en/docs/Writing policies/variables.md

```diff
@@ -111,13 +145,12 @@ busybox   0/1     Pending   0          25m   created-by=kubernetes-admin,run=bus
 
 In the output, we can clearly see the value of our `created-by` label is `kubernetes-admin` which, in this case, is the user who created the Pod.
 
-## Variables from ConfigMap resources
 
-A [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) resource in Kubernetes is often a source of configuration details which can be consumed by applications. This data can be written in multiple formats, stored in a Namespace, and accessed easily. Kyverno supports using a ConfigMap as a data source for variables, either as key/value pair data or multi-line strings. When a policy referencing a ConfigMap resource is evaluated, the ConfigMap data is checked at that time ensuring that references to the `ConfigMap` are always dynamic. Should the ConfigMap be updated later, subsequent policy lookups will pick up the data at that point.
+## Variables from external data sources
```

Comment:
Little confused here as it seems you've picked up the whole ConfigMap section and transported it to a new external data sources page, but you've left this bit as a kind of redirect link?

Reply:
It seemed to me, that we need discussion in the variables section to indicate that its possible to define variables from external data sources. Do you see a better way to organize the two chapters?

Reply:
I moved the example...